src/Jobs/Recruit.Vacancies.Jobs/SpikeJob/SpikeJob.cs

```diff
@@ -0,0 +1,34 @@
+using System;
```

Comment:
Do you want to include this file?

Reply:
Yeah, I think it will be useful since I seem to create it so many times for debugging purposes.